ios-base/DroidKaigi 2020/Announcements/ViewModels/AnnouncementsViewModel.swift

```diff
@@ -0,0 +1,54 @@
+import Foundation
+import ios_combined
+import RxCocoa
+import RxSwift
+
+protocol AnnouncementsViewModelType {
+    // Output
+    var announcements: Driver<[Announcement]> { get }
+    var error: Driver<KotlinError?> { get }
+
+    // Input
+    func viewDidLoad()
+    func pullToRefresh()
+}
+
+final class AnnouncementsViewModel: AnnouncementsViewModelType {
+    let announcements: Driver<[Announcement]>
+    let error: Driver<KotlinError?>
+
+    private let viewDidLoadReplay = PublishRelay<Void>()
```

Comment:
typo.
Please change to `private let viewDidLoadRelay = PublishRelay<Void>()` 🙏

Reply:
I fixed.

https://github.com/DroidKaigi/conference-app-2020/pull/628/commits/d54f4de55bbecf433b45aa0ca24bc6f98f483ac8